TINY-9680: context toolbar not showing the correct status for `advlist` (#8591)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- Table toolbar was visible even if the table was within a noneditable host element. #TINY-9664
- Quickbar toolbars was shown for noneditable contents in a noneditable root. #TINY-9460
- Inline alert in the "Search and Replace" dialog persisted when it wasn't necessary. #TINY-9704

> - Context toolbars was not showning the correct status for the `advlist` buttons. #TINY-9680

Reviewer: ```suggestion
- Context toolbars displayed the incorrect status for the `advlist` plugin buttons. #TINY-9680
```
